java/client/src/org/openqa/selenium/virtualauthenticator/VirtualAuthenticatorOptions.java

```diff
@@ -0,0 +1,98 @@
+// Licensed to the Software Freedom Conservancy (SFC) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The SFC licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package org.openqa.selenium.virtualauthenticator;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Options for the creation of virtual authenticators.
+ * @see http://w3c.github.io/webauthn/#sctn-automation
+ */
+public class VirtualAuthenticatorOptions {
+
+  enum Protocol {
+    CTAP2("ctap2"), U2F("ctap1/u2f");
+
+    public final String id;
+
+    private Protocol(String id) {
+      this.id = id;
+    }
+  }
+
+  enum Transport {
+    USB("usb"), NFC("nfc"), BLE("ble"), INTERNAL("internal");
```

Comment:
Are these ordered in this way to ensure the ordinals match up with something in the spec? If not, it would be lovely if these were ordered alphabetically.